.github/workflows/main.yml

```diff
@@ -110,11 +110,13 @@ jobs:
         run: mvn -B package -DskipTests=true
       - name: Set up CockroachDB
         run: |
-          wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.6.linux-amd64.tgz | tar  xvz
-          cd cockroach-v20.2.6.linux-amd64/ && ./cockroach start-single-node --insecure &
+          wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.8.linux-amd64.tgz | tar  xvz
```

Comment:
I think something went wrong when rewriting the commit history here. Could you take a look at that? Besides this, the PR looks ready to be merged.

Reply:
solved